pilot/spec/manager_spec.rb

```diff
@@ -490,11 +490,6 @@
           before(:each) do
             expect(UI).to receive(:verbose).with("App Platform (#{fake_app_platform})")
             fake_manager.instance_variable_set(:@config, { pkg: fake_pkg })
-
-            allow(FastlaneCore::PkgFileAnalyser)
-              .to receive(:fetch_app_platform)
-              .with(fake_pkg)
-              .and_return(fake_app_platform)
```

Comment:
Removing this causes the test to fail when there is no `FastlaneCore::PkgFileAnalyser.fetch_app_platform` method.

Reply:
I think this test needs to be added back in? I also think that this (and the ipa) test should replace `allow` with `expect` 😇  That would probably have caught this error before going out 😅

Reply:
Great, thanks for pointing that out! I've made the change.